miso-web/src/main/webapp/scripts/freezer_ajax.js

```diff
@@ -341,8 +390,8 @@
     var table = $('#levelTwoStorageLayout');
     var row = $('<tr>');
     var cell = $('<td>');
-    for(box in storage.boxes){
-    	cell.append(document.createTextNode(storage.boxes[box].alias)).append('<br/>');
+    for (box in storage.boxes) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "box" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)